error starting Homebridge .4.50

Hi,

I have this issue:

[2019-9-13 18:21:12] Loaded config.json with 0 accessories and 1 platforms.
[2019-9-13 18:21:12] ---
[2019-9-13 18:21:12] Loaded plugin: homebridge-camera-ui
module.js:550
    throw err;
    ^

Error: Cannot find module 'form-data'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/lib/node_modules/homebridge-camera-ui/src/accessories/camera.js:14:18)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)

--- a/package.json
+++ b/package.json
@@ -48,6 +48,7 @@
     "helmet": "^3.18.0",
     "tcp-ping": "^0.1.1",
     "basic-ftp": "^3.6.0",
-    "is-image": "^3.0.0"
+    "is-image": "^3.0.0",
+    "form-data": "^2.5.1"
   }
 }

